quora-api/src/main/java/com/upgrad/quora/api/Controller/UserController.java

```diff
@@ -0,0 +1,69 @@
+package com.upgrad.quora.api.controller;
+
+
+import com.upgrad.quora.api.model.SigninResponse;
+import com.upgrad.quora.api.model.SignoutResponse;
+import com.upgrad.quora.api.model.SignupUserRequest;
+import com.upgrad.quora.api.model.SignupUserResponse;
+import com.upgrad.quora.service.business.SignUpBusinessService;
+import com.upgrad.quora.service.entity.UserAuthEntity;
+import com.upgrad.quora.service.entity.UserEntity;
+import com.upgrad.quora.service.exception.AuthenticationFailedException;
+import com.upgrad.quora.service.exception.SignOutRestrictedException;
+import com.upgrad.quora.service.exception.SignUpRestrictedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Base64;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/")
```

Comment:
This is not necessary as you have added ```@RequestMapping``` at each method level. This will give wrong api path.

Reply:
OK I will removed it.